### PR TITLE
new map!

### PR DESCRIPTION
**Pull request by starlighthowls**
**Changelog**

:cl:
- add: New 0-12 pop map Skimmer
- fix: pneumatic cannon makes the silly noise now 
 
Skimmer-0

### CODE REVIEW

**Comment by joelsgp** [comment]
it doesn't like that Skimmer.yml is uppercase. must need to be skimmer.yml. looks great, very small industrial mining vessel orbiting close over a planet vibes

P.S. the way prs work now is each one needs to be approved by any member other than the person who created it, then it can be merged by anyone

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/pneumatic_cannon.yml [hunk 1]
```diff
@@ -30,6 +31,7 @@
   - type: ContainerAmmoProvider
     container: storagebase
   - type: PneumaticCannon
+    throwItems: false
```
Comment (Tiger-Caldwell) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/pneumatic_cannon.yml:34]: This turns the base improvised pneumatic cannon into a gun that can fire bullets.

**Review by Tiger-Caldwell — CHANGES_REQUESTED** [review]
- The particle accelerator power box accepts MV cable, but there is only an HV cable attached to it. This would be a huge headache for engineers at round start.
- It would be nice if the APCs and substations were labeled by what area/department they power in the power monitor
- Cameras don't seem to be working on the camera monitor

**Comment by Darkmajia** [comment]
the pneumatic cannon changes are a little beyond just "makes the silly noise", please try to keep separate changes in separate pull requests

**Comment by starlighthowls** [comment]
> the pneumatic cannon changes are a little beyond just "makes the silly noise", please try to keep separate changes in separate pull requests

i was trying to fix a previous change to swap the fire noise id made bcs it hadnt worked correctly it seems it will only play the noise i want it to if i mess with the firing settings?? its really weird
ill make dif prqs in future tho!

**Comment by starlighthowls** [comment]
> * The particle accelerator power box only accepts MV cable, but there is an HV cable attached to it. This would be a huge headache for engineers at round start.
> 
>     * It would be nice if the APCs and substations were labeled by what area/department they power in the power monitor
> 
>     * Cameras don't seem to be working on the camera monitor

ah i got the wire type wrong for the PA thanks for pointing that out and ill label the power stuff! 
with the cameras im not sure why they wouldnt be working.. did you try refreshing subnets?

**Pull request closed without merging**